js/bybit.js

```diff
@@ -4702,6 +4703,57 @@ module.exports = class bybit extends Exchange {
         return await this[method] (this.extend (request, params));
     }
 
+    async setPositionMode (hedged, symbol = undefined, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' setPositionMode() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        if (market['settle'] === 'USDC') {
+            throw new NotSupported (this.id + ' setPositionMode() does not support market ' + symbol + '');
+        }
+        if (market['inverse'] && !market['future']) {
+            throw new BadRequest (this.id + ' setPositionMode() must be inverse future');
+        }
+        const request = {
+            'symbol': market['id'],
+        };
+        let method = undefined;
+        let mode = '';
```

Comment:
Let's not initialize `mode` to an empty string here — in all such cases, please, set `undefined` instead.

```suggestion
        let mode = undefined;
```